### ISSUE TITLE
Getting Command "link-caa" not found when setting up local dev environment

### ISSUE THREAD

**Issue by aedrianescultura** [aws-amplify/amplify-cli#5486]
**Describe the bug**
Getting this error when running npm run setup-dev-win

```
error Command "link-caa" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amplify-cli@0.1.0 setup-dev-win: `yarn dev-build && yarn link-win && yarn link-caa`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amplify-cli@0.1.0 setup-dev-win script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Amplify CLI Version**
amplify-cli master branch

**To Reproduce**
yarn config set workspaces-experimental true
npm run setup-dev-win

**Expected behavior**
Should finish without errors

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Node Version. v12.16.3

**Additional context**
Modifying package.json and updating setup-dev-win with line below fixes the issue

`"setup-dev-win": "lerna exec -- del /f package-lock.json && lerna bootstrap && cd packages/amplify-cli && del /f package-lock.json && npm link && cd ../.. && lerna run build",`

**Comment by aedrianescultura** [comment]
@nikhname I tried **link-aa-dev** but it fired the ln command which is not available in Windows.

```
$ node ./scripts/link-bin.js packages/amplify-cli/bin/amplify
Done in 2.10s.
yarn run v1.22.4
$ cd packages/amplify-app && ln -s $(pwd)/bin/amplify-app $(yarn global bin)/amplify-app-dev && cd -
'ln' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! amplify-cli@0.1.0 setup-dev-win: `yarn dev-build && yarn link-win && yarn link-aa-dev`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the amplify-cli@0.1.0 setup-dev-win script.
```

**Comment by nikhname** [comment]
@aedrianescultura yes, we need to use the link-bin script that works with Windows. I'm updating the PR. Thanks!

**Comment by aedrianescultura** [comment]
@nikhname link-bin seems designed to create amplify-dev.cmd only so link-aa-win will overwrite link-win.  
We can make link-bin.js accept another parameter for the command name like the following:

In package.json
```
    "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
    "link-aa-win": "node ./scripts/link-bin.js packages/amplify-app/bin/amplify-app amplify-app-dev",
    "setup-dev-win": "yarn dev-build && yarn link-win && yarn link-aa-win",
```

In link-bin.js
```
if (process.argv.length < 4) {
  console.log('requires 2 arguments. source');
  process.exit(1);
}

const src = path.join(process.cwd(), process.argv[2]);
let dest;
if (process.argv.length === 4) {
  const yarnGlobalBin = childProcess
    .execSync('yarn global bin')
    .toString()
    .trim();
  dest = path.join(yarnGlobalBin, process.argv[3]);
} else {
  dest = path.isAbsolute(process.argv[4]) ? process.argv[4] : path.join(process.cwd(), process.argv[3]).toString();
  dest = path.join(yarnGlobalBin, process.argv[4]);
}
```

**Comment by edwardfoyle** [comment]
The fix for this has been released in CLI v 4.32.0
